src/org/usfirst/frc/team694/robot/subsystems/Lift.java

```diff
@@ -76,12 +78,8 @@ public void toggleBrake() {
         }
     }
 
-    public void goUp() {
-        leftLiftMotor.set(1);
-    }
-
-    public void goDown() {
-        leftLiftMotor.set(-1);
+    public void setLiftMotors(double speed) {
```

Comment:
I'd suggest you undo this and use "goUp" and "goDown". We are either going at 100% speed (confirm with engineering) or 0% speed. We do not want our user to have the option to set the lift to move at anything in between